### PR TITLE
Smoothly attach assessment reports to systems via the API

### PR DESCRIPTION
**Pull request by gregelin**
This PR enables the smooth, automated attachment of assessment summary reports (in a particular format) to individual GovReady-Q project systems and to specific system deployments (e.g., Dev, Prod).

PR also includes a sample System Assessment Result (SAR) middleware pipeline in `tools/simple_sar_server` that includes a simple SAR service generating sample SAR summaries (`sar.py`) and SAR ETL service to transform and submit the SAR to GovReady via authenticated API (`sar_etl.py`).

Assessment handling

### CODE REVIEW

File: controls/views.py [hunk 1]
```diff
@@ -3048,7 +3048,8 @@ def view_system_assessment_result_summary(request, system_id, sar_id=None):
     project = system.projects.all()[0]
     sar = get_object_or_404(SystemAssessmentResult, pk=sar_id) if sar_id else None
 
-    sar_items = [item for item in sar.assessment_results] if sar.assessment_results != None else []
+    # Get assessment targets results from wrapped SAR data
+    sar_items = [item for item in sar.assessment_results['sar']] if sar.assessment_results != None else []
```
Comment (davidpofo) [inline controls/views.py:3052]:
```suggestion
    sar_items = [item for item in sar.assessment_results['sar']] if sar.assessment_results != None else []]
```

**Review by davidpofo — COMMENTED** [review]
need greg PR party 04/08

**Review by alexanderward — DISMISSED** [review]
David has a change on line 3052 to be resolved.

File: controls/views.py (hunk 1 above)
Comment (gregelin) [reply controls/views.py:3053]: @davidpofo Take a closer look at the first line. The goal is to return an array or and empty array. First line is correct. Suggestion has incorrect syntax.

**Review by davidpofo — APPROVED** [review]
lgtm pr party 4/22

**Pull request merged**